Задание для кандидатов. Выполнить ревью.sql

```diff
@@ -0,0 +1,123 @@
+create procedure syn.usp_ImportFileCustomerSeasonal
+	@ID_Record int
+AS
+set nocount on
+begin
+	declare @RowCount int = (select count(*) from syn.SA_CustomerSeasonal)
+	declare @ErrorMessage varchar(max)

```

Comment:
Комментарий 2
Для объявления переменных declare используется один раз
```suggestion
	 , @ErrorMessage varchar(max)

```